test/app.js

```diff
@@ -62,13 +64,15 @@ test('test input focus', async t => {
   buttons = await app.client.elements('button');
   await app.client.elementIdClick(buttons.value[3].ELEMENT);
 
-  await sleep(1000);
+  await sleep(2000);
```

Comment:
Are you increasing numbers just to be sure or the i18next performance is lower?

Reply:
AppVeyor build failed. I'll check the performance.